projects/storefrontlib/src/shared/components/order-overview/order-overview.component.spec.ts

```diff
@@ -107,6 +107,11 @@ const mockOrder: Order = {
   },
 };
 
+const mockFormattedAddress: string[] = [
+  'test1, , test3, test4',
```

Comment:
This one is an unformatted address I think. In general, it's a bit misleading that you have this array as such. I'd rather have two arrays or two separate variables (strings) and then make them an array. It would make the test more clear.